dwellingly/src/views/addProperty.js

```diff
@@ -0,0 +1,162 @@
+                                                value={values.city}
+                                                placeholder="Portland"
+                                            />
+                                            {errors.city ? (<div className="error-message">{errors.city}</div>) : null}
+                                        </div>
+                                        <div className="form-row columns">
+                                            <label className="column is-one-fifth" htmlFor="state">State</label>
+                                            <Field
+                                                className="column form-field"
+                                                type="text"
+                                                name="state"
+                                                onChange={handleChange}
+                                                value={values.state}
+                                                placeholder="OR"
+                                            />
+                                            {errors.state ? (<div className="error-message">{errors.state}</div>) : null}
+                                        </div>
+                                        <div className="form-row columns">
+                                            <label className="column is-one-fifth" htmlFor="zipcode">Zipcode</label>
+                                            <Field
+                                                className="column form-field"
+                                                type="text"
+                                                name="zipcode"
+                                                onChange={handleChange}
+                                                value={values.zipcode}
+                                                placeholder="97217"
+                                            />
+                                            {errors.zipcode ? (<div className="error-message">{errors.zipcode}</div>) : null}
+                                        </div>
+                                        <div className="form-row columns">
+                                            <label className="column is-one-fifth" htmlFor="zipcode">Units</label>
+                                            <Field
+                                                className="column form-field"
+                                                type="text"
+                                                name="units"
+                                                onChange={handleChange}
+                                                value={values.zipcode}
+                                                placeholder="10"
+                                            />
+                                        </div>
+                                        <div className=" add-property__assign-manager-container">
```

Comment:
Like we talked about, let's comment this out and make a new issue for it. Be sure to mention the `react-select` package in the issue.
It'll be waiting on #120 to get done, then we can start using that api endpoint.